src/containers/Application/ElementsBox.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
```

Comment:
This rename is going to cause conflicts with my PR #34 -- a straight merge will just have two files (ElementsBox and ElementsArea) with different content (since I edited that one). I should probably rename mine to ElementsBox I guess, but I might wait till this is merged first? Let me know what you think and the best way to deal with it.

(Same for ApplicationPage -> ApplicationPageWrapper now that I think of it)

Reply:
I think the git diff would sort that one out. You can try doing the following -> merge this pr with yours to check. It would probably not complain about those things. We can do that tomorrow to check together if that's better.

Reply:
I don't think it will specify conflicts -- it'll just end up keeping both files, wouldn't it? Anyway, we can check this in the morning when we merge. :)

Reply:
> I don't think it will specify conflicts -- it'll just end up keeping both files, wouldn't it?

Yes. Like you said, you have added changes to the original one, so it git wouldn't know how to resolve this. I ended up having to do the merging manually and then remove the old `ElementsArea` file.

Reply:
should use `git mv` when renaming like this